concourse/pipelines/windows-image-build.jsonnet

```diff
@@ -819,6 +843,10 @@ local ImgGroup(name, images, environments) = {
              [
                common.GcsImgResource(image, 'windows-install-media')
                for image in windows_install_media_images
+             ] +
+             [
+               common.GcsShasumResource(image, 'windows-install-media')
```

Comment:
Did you intend to make this the shasum?

Reply:
I added it so it passes the presubmit, however it doesn't have any function.

The windows media images don't use the sha-sum or sbom, however it uses the "imgpublishjob" which is common for all the windows builds.

Reply:
Fixed: the current way skips the shasum steps for windows media images by passing in "generate_shasum = false"